examples/snippets/filesystem/cloud/s3_upload.py

```diff
@@ -0,0 +1,17 @@
+"""
+An example script to demonstrate the use of the s3 function to upload file
+"""
+
+import tamr_toolbox as tbox
+from boto3.session import Session
+
+S3Client = Session("credentials").client("s3")
```

Comment:
This works, but is not what is recommended in the boto3 docs [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-uploading-files.html).  Instead I see:
```suggestion
import boto3

s3_client = boto3.client("s3")
```